<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchPhotoDialog</class>
 <widget class="QDialog" name="searchPhotoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cercador de Fotos</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listPhotos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxCategory">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Categoria</string>
   </property>
   <widget class="QRadioButton" name="radioPersona">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Persona</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioLloc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Lloc</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Altres</string>
    </property>
   </widget>
   <widget class="QLabel" name="MetadataLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Metadata:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMetadata">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CercaMetadataBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxFoto">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Foto:</string>
   </property>
   <widget class="QLabel" name="labelTitol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Titol:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTitol">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cercaFotoBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OpenBtn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>590</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Obre</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Albums</string>
   </property>
   <widget class="QComboBox" name="comboAlbums">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="eliminarBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editarBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>590</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Desar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitol_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTitol2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFitxer">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fitxer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Direcci√≥ del Fitxer</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listMetadataP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listMetadataL">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>461</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listMetadataA">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioPersona2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Persona</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioLlocs2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Llocs</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioAltres2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Altres</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMetadata2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Metadata:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMetadata2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addMetadataBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delMetadataBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Treure</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>searchPhotoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>searchPhotoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
